Werkgroep API design rules/naamgeving.md

```diff
@@ -0,0 +1,56 @@
+# Naamgeving
+## Generieke naamconventies
+### Speciale tekens
```

Comment:
Wat wordt bedoeld met "Generieke naamconventies"? In welke zin generiek?

Deze 2 bullets zijn bijv. van toepassing voor:
* URIs (path/query/header namen).
* JSON property keys

Dit moeten we ergens concreet maken. Verderop hebben we het namelijk over enumeraties; dat zegt weer iets over de waardes ipv keys.

Reply:
Ik heb de tekst van de extensies zelf aangepast als onderdeel van deze PR. Laat maar weten of die concreet genoeg zijn nu, of verdere concretisering behoeven.